src/pages/index.js

```diff
@@ -66,8 +66,8 @@ export default function Home() {
   const { siteConfig } = useDocusaurusContext();
   return (
     <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Learn about Cloudberry Database in just a few minutes."
+      title={`Next generation unified database for analytics and AI`}
```

Comment:
If it is a title of a website, title case capitalization is better.
```suggestion
      title={`Next-Generation Unified Database for Analytics and AI"`}
```

Reply:
Both ways are okay, so keep it unchanged. Thanks.